packages/runtime/src/widgets/Dropdown.tsx

```diff
@@ -39,6 +63,41 @@ const Dropdown: React.FC<DropdownProps> = (props) => {
     },
     [action],
   );
+
+  const { namedData } = useTemplateData({
+    data: itemTemplate?.data,
+    name: itemTemplate?.name,
+  });
+
+  const options = useMemo(() => {
+    if (values?.items) {
+      return values.items.map((item) => {
+        return (
+          <Select.Option key={item.value} value={item.value}>
+            {item.label}
+          </Select.Option>
+        );
+      });
+    }
+
+    if (isObject(itemTemplate)) {
+      return namedData.map((item: unknown, index) => {
+        return (
+          <Select.Option
+            key={index}
+            value={evaluate(defaultScreenContext, itemTemplate.value, {
+              [itemTemplate.name]: get(item, itemTemplate.name) as unknown,
+            })}
```

Comment:
@evshi I need your help here,

as right now I am getting the error of 
```Unsafe assignment of an `any` value```

and If I wrap this with either Number or String, there is no error.
but we can't be sure that every time we will get Number or String as a value.

Reply:
`evaluate` takes a type argument that will case the return value